src/components/books/book/book.js

```diff
@@ -1,12 +1,21 @@
 /* eslint-disable react/prop-types */
```

Comment:
Kindly remove the disabled Linters, Linting helps in making your code cleaner and efficient ✊ 

Please check for other occurrences 👍